applications/datalinker/values-idfdev.yaml

```diff
@@ -1,2 +1,3 @@
 config:
   separateSecrets: true
+  overrideButlerRepositoryIndex: "s3://butler-us-central1-repo-locations/data-dev-repos-remote.yaml"
```

Comment:
There's some minor stupidity here -- downloading this index file requires the S3 credentials we're trying to get rid of. 🤦 

The new butler factory thing no longer requires this configuration to be in a file -- you can just pass in a dict of labels and URIs.  So I think it makes sense for the content of this file to be in phalanx instead.  When we go in to get rid of all the credentials I'll add an environment-level global configuration for this and pass the contents in an environment variable instead of using a URI.